pages/css/components/blankslate.md

```diff
@@ -15,8 +15,11 @@ Wrap some content in the outer `.blankslate` wrapper to give it the blankslate a
 
 ```html
 <div class="blankslate">
-  <h3>This is a blank slate</h3>
-  <p>Use it to provide information when no dynamic content exists.</p>
+  <img src="https://ghicons.github.com/assets/images/light/Pull%20Request.png" alt="Pull requests" class="mb-3">
+  <h3 class="mb-1">You don’t seem to have any pull requests.</h3>
+  <p>Pull requests help you discuss potential changes before your changes are merged into the base branch.</p>
+  <button class="btn btn-primary my-3" type="button">New pull request</button>
+  <p><button class="btn-link" type="button">Learn more</button></p>
```

Comment:
```suggestion
  <p><button class="btn-link">Learn more</button></p>
```

Reply:
Button elements don't need `type='button'` on them

Reply:
@emplums good to know! I just grabbed the code from our docs so we should remove it there too.